akka-http/src/main/resources/reference.conf

```diff
@@ -55,5 +55,11 @@ akka.http {
 
     # The maximum size for parsing lines of a server-sent event.
     max-line-size = 4096
+
+    # Empty events are used by the akka-http SSE infrastructure for heartbeats. Received heartbeats are ignored by default
+    # and not delivered in the user stream.
+    #
+    # This setting can be enabled to pass those empty events to the application for explicitly handling.
+    emit-empty-events = off
```

Comment:
hmm, so if this is 'off' they are still emitted over the wire, just not passed to the application? That's not super clear from the name, but I'm not sure I have a better suggestion either.

Reply:
Yeah, I hoped that `emit` was a stream term vs. network terms.

Reply:
Yeah, I hoped that `emit` was a stream term vs. network terms.

Reply:
Github was lagging...